docs/content/en/tasks/secure/encryption/_index.md

```diff
@@ -0,0 +1,193 @@
+* K8ssandra version 2.0 or later
+* An SSL encryption store, as covered in the next section
+
+## Generating SSL encryption stores
+
+If you do not have a set of encryption stores available, follow the instructions in [this TLP blog post](https://thelastpickle.com/blog/2021/06/15/cassandra-certificate-management-part_1-how-to-rotate-keys.html). More specifically, use [the script](https://github.com/thelastpickle/cassandra-toolbox/tree/main/generate_cluster_ssl_stores) used to generate the SSL stores.
+
+You could clone the [cassandra-toolbox](https://github.com/thelastpickle/cassandra-toolbox) GitHub repository, and create a `cert.conf` file with the following format:
+
+```conf
+[ req ]
+distinguished_name     = req_distinguished_name
+prompt                 = no
+output_password        = MyPassWord123!
+default_bits           = 2048
+
+[ req_distinguished_name ]
+C                      = FR
+ST                     = IDF
+L                      = Paris
+O                      = YourCompany
+OU                     = SSLTestCluster
+CN                     = SSLTestClusterRootCA
+emailAddress           = youraddress@whatever.com
+```
+
+Next, run:
+
+```bash
+./generate_cluster_ssl_stores.sh -v 10000 -g cert.conf
+```
+
+The `-v` value above sets the validity of the generated certificates in days. 
+
+{{% alert title="Tip" color="success" %}}
+Don't set this `-v` days value too low. Doing so would require you to rotate the certificates too often; it's not a trivial operation.
+{{% /alert %}}
+
+The command output should be a folder containing a keystore, a truststore, and a file containing their respective passwords.
+
+Rename the keystore file to `keystore`, and rename the truststore file to `truststore`. Then create a Kubernetes secret with the following command:
+
+```bash
+kubectl create secret generic server-encryption-stores --from-file=keystore --from-literal=keystore-password=<keystore password> --from-file=truststore --from-literal=truststore-password=<truststore password> -o yaml > server-encryption-stores.yaml
+```
+
+Replace the `<keystore password>` and `<truststore password>` above with each store's actual password.
+
+{{% alert title="Tip" color="success" %}}
+You can repeat the above procedure to generate encryption stores for client-to-node encryption, changing the secret name appropriately.
+{{% /alert %}}
+
+## Creating a cluster with internode encryption
+
+In order to create a K8ssandra cluster with encryption, first create a namespace and the encryption stores secrets previously generated in it.
+
+In the `K8ssandraCluster` manifest, you will need to configure encryption settings in the `config/cassandraYaml` section.
+
+Also, you'll need to reference the encryption stores' secrets under:
+
+* `cassandra/serverEncryptionStores` 
+* *Or* `cassandra/clientEncryptionStores`
+
+We emphasized "or" here because, theoretically, the keystore and truststore could be placed in different secrets.
```

Comment:
server encryption and client encryption are different things and they both have their own keystore/truststore pair.
The "or" here shows that you can turn on either independently, or both.
Server is for internode communications encryption, and client is for client to node communications encryption.

It is true that truststore and keystore can be in different secrets, but its unrelated to the type of encryption we're dealing with.

Reply:
Inc'd comments - thanks Alex.